packages/babel-core/src/transformation/file/options/option-manager.js

```diff
@@ -117,10 +117,14 @@ export default class OptionManager {
       let plugin, options;
 
       // destructure plugins
-      if (Array.isArray(val)) {
-        [plugin, options] = val;
+      if (val) {
```

Comment:
Should we also check `typeof val`?  I think it should always be either "Array", `'string'` or `'function'`.